test/source/tests/settings.ts

```diff
@@ -506,6 +506,20 @@ export const defineSettingsTests = (testVariant: TestVariant, testWithBrowser: T
       Util.deleteFileIfExists(downloadedAttachmentFilename);
     }));
 
+    ava.default('settings - check if downloaded attachment name is correct', testWithBrowser('compatibility', async (t, browser) => {
+      // `what's up?.txt` becomes `what's_up_.txt` and this is native way and we can't change this logic
+      // https://github.com/FlowCrypt/flowcrypt-browser/issues/3505#issuecomment-812269422
+      const downloadedAttachmentFilename = `${__dirname}/what\'s_up_.txt`;
+      const inboxPage = await browser.newPage(t, TestUrls.extension(`chrome/settings/inbox/inbox.htm?acctEmail=flowcrypt.compatibility@gmail.com&threadId=1821bf879a6f71e0`));
+      // @ts-ignore
+      await (inboxPage.target as Page)._client.send('Page.setDownloadBehavior', { behavior: 'allow', downloadPath: __dirname });
```

Comment:
I'd very much like to re-use the method `awaitDownloadTriggeredByClicking` that we have in `controllable.ts`, rather than making copies this low-level code.
We could make `awaitDownloadTriggeredByClicking` return filenames as well and probably an option to include the file contents (so we can turn it off when it's not required)

Reply:
Thank you for your suggestion, That method seems what I was looking for.